### ISSUE TITLE
openpilot failed to build

### ISSUE THREAD

**Issue by andyroo-t** [jc01rho-openpilot-BoltEV2019-KoKr/boltpilot#247]
콤마3 이슈 외에 제보를 받고있지 않습니다.
please post only comma3 issues.

**Describe the bug**
Attempt to load v0.819-c3 fails to build at virgin install

**To Reproduce**
1. Load from repository
2. Accept OS update
3. Screen errors out with failure to build
4. See screenshot of build error here:

https://i.postimg.cc/dtZjKDks/4-BC9-DC0-A-ADB2-432-C-8-E84-8-D9-BCA858-D3-F.jpg

**Expected 

Clean boot and build of 0819

**Additional context**

**Comment by jc01rho** [comment]
v0.819-c3 is kinda old tag, I'd recommend release or develop. how about use release?

**Comment by andyroo-t** [comment]
Hi,

Frankly, I think the UI on v.819 is more pleasing and cleaner and for some reason it seems to drive better.

**Comment by jc01rho** [comment]
Okay. But there is no change there recently.

I don't know why fails, and I have no time to fix or try 0819 it. Needs some time to fix it.

In first, revert the latest changes. But i cannot sure it fixes.

**Comment by andyroo-t** [comment]
A revert is better than nothing for that release...

**Comment by jc01rho** [comment]
revert done. uploading in progress.

reverted  changes in 2023-Aug, current latest changes are same as 2022-April.

**Comment by andyroo-t** [comment]
Thank you for taking the time to do that.

I'll check it in a few hours or less.

cheers,

andy

**Comment by andyroo-t** [comment]
Hi,

It's forcing an OS update during install, unlike the original April build, which then causes camera errors where the camera won't start. 

Is there any way to turn that OS update off and have it run off your reverted build?

**Comment by jc01rho** [comment]
c3 checks version currently installed os versus source codes os,
when it differs, run updates.

I don't know why camera errors, anyway.

set the os version follows 2023-Aug.

this is the only option I can try :)

**Comment by jc01rho** [comment]
pushed,
https://github.com/jc01rho-openpilot-BoltEV2019-KoKr/boltpilot/commit/0c03045eee4042fa759335e83e03896a36df693d

**Comment by andyroo-t** [comment]
The build is for Hyundai/Kia is the problem. 

Bolt is not available to select as the car, like it was in May/June. This is why the camera errors.

**Comment by jc01rho** [comment]
okay, so, Are you  still facing that  unavailable  car-bolt selection?

**Comment by andyroo-t** [comment]
Yes, I just redownloaded and I get a list of Hyundai & Kia cars, no Bolt.

**Comment by andyroo-t** [comment]
1:30 am here, so I won't be getting to another reload until morning...maybe 10 hours from now. If you have time to look into it by then.

thank you,

andy
